mobsf/MobSF/views/saml2.py

```diff
@@ -127,10 +127,19 @@ def get_redirect_url(req):
 
 def get_user_role(roles):
     """Get user role."""
-    mrole = any(MAINTAINER_GROUP.lower() in gp.lower() for gp in roles)
-    if mrole:
+    maintainer_groups = [g.lower() for g in settings.IDP_MAINTAINER_GROUP]
+    viewer_groups = [g.lower() for g in settings.IDP_VIEWER_GROUP]
+    user_roles = [gp.lower() for gp in roles]
+    mrole = any(gp in maintainer_groups for gp in user_roles)
+    vrole = any(gp in viewer_groups for gp in user_roles)
+    if mrole or settings.IDP_MOBSF_DEFAULT_GROUP == 'Maintainer':
+        logger.info('User assigned to %s group.', MAINTAINER_GROUP)
         return MAINTAINER_GROUP
-    return VIEWER_GROUP
+    elif vrole or settings.IDP_MOBSF_DEFAULT_GROUP == 'Viewer':
+        logger.info('User assigned to %s group.', VIEWER_GROUP)
+        return VIEWER_GROUP
+    logger.warning('User does not have an authorized SSO group.')
```

Comment:
Is there a reason to support this? We do not want to over complicate SSO roles. MobSF only has two role privileges a read-only default role for everyone and a read-write maintainer role for maintainer group.